doc/conf.py

```diff
@@ -310,6 +310,10 @@
 # details
 numpydoc_show_class_members = False
 
+# Scraper, copied from https://github.com/mne-tools/mne-python/
+from nilearn.reporting import _ReportScraper
+report_scraper = _ReportScraper()
+scrapers = ('matplotlib', report_scraper)
```

Comment:
You created the `scrapers` variable but did not use it to modify the SG config, see:

https://sphinx-gallery.github.io/configuration.html#image-scrapers

Add a `'image_scrapers': scrapers,` up around line 304 in the `sphinx_gallery_conf` dict (after moving these lines above so they precede the creation of this dict) and it should work.